02_julia_intro/02_julia_intro.jl

```diff
@@ -547,10 +552,42 @@ begin
 	scatter!(sequence, label=false, color="purple", size=(450, 300))
 end
 
-# ╔═╡ 11f33ecc-544f-11eb-35d5-27a280cdce1b
+# ╔═╡ 53a0950c-6d79-11eb-0914-4de1f259e95e
+md"
+In the example above, a plot is created when we call the ```plot()``` function. What the ```scatter!()``` call then does, is to modify the global state of the plot in-place. If not done this way, both plots wouldn't be sketched together.
+
+A nice feature that the Plots.jl package offers, is the fact of changing plotting backends. There exist various plotting packages in Julia, and each one has its own special features and aesthetic flavour. The Plots.jl package integrates these plotting libraries and acts as an interface to communicate with them in an easy way.
+By default, the ```GR``` backend is the one used. In fact, this was the plotting engine that generated the plots we have already done. The most used and maintained plotting backends up to date, are the already mentioned ```GR```, ```Plotly/PlotlyJS```, ```PyPlot```, ```UnicodePlots``` and ```InspectDR```.
+The backend you choose will depend on the particular situation you are facing. For a detailed explanation on backends, we recommend you visit the Julia Plots [documentation](https://docs.juliaplots.org/latest/backends/). Through the book we will be focusing on the ```GR```backend, but as a demonstration of the ease of changing from one backend to another, consider the code below.
+The only thing added to the code for plotting that we have already used, is the ```pyplot()``` call to change the backend. If you have already coded in Python, you will feel familiar with this plotting backend. 
```

Comment:
The only thing added to the code that we have already used is the ```pyplot()``` call to change the backend. If you have experience coding in Python, you may feel familiar with this plotting backend.